src/utils/hs_logger.R

```diff
@@ -1,12 +1,4 @@
-box::use(logger[log_threshold, log_info, log_debug])
-
-#' Sets up the logging environment
-#'
-#' @export
-configure_logger <- function() {
-  log_level <- Sys.getenv("LOG_LEVEL", unset = "INFO")
-  log_threshold(log_level)
-}
+box::use(logger[log_info, log_debug])
```

Comment:
Just while we're here, I noticed in `monitoring_log_setup()` that `FIRST_RUN` should be `HS_FIRST_RUN`, so maybe can just put that fix into this PR.